docs/modules/ROOT/pages/nre.adoc

```diff
@@ -0,0 +1,32 @@
+= NRE and Scripting
+
+Nile provides a Runtime Environment for scripting that you can use for multiple tasks. The Script is any python file implementing an asynchronous `run(nre)` function to receive a NileRuntimeEnvironment object exposing Nile's scripting API.
+
+== Running a script
+
+. Create a file named *test_script.py* in the root directory of your project.
```

Comment:
i actually suggest having a `scripts/` folder. the 3rd instruction below also asumes a generic path. we should eventually add one to `nile init`'s sample project.